config/spotbugs-exclude.xml

```diff
@@ -175,4 +175,11 @@
         <Method name="processFiltered"/>
         <Bug pattern="NP_GUARANTEED_DEREF_ON_EXCEPTION_PATH"/>
     </Match>
+    <Match>
+        <!-- We cannot modify fields since we won't have the values
+                to create the instance until the setters are called -->
+        <Class name="com.puppycrawl.tools.checkstyle.filters.SuppressionXpathSingleFilter"/>
+        <Method name="accept"/>
+        <Bug pattern="UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR"/>
+    </Match>
```

Comment:
please append such case to existing group - https://github.com/checkstyle/checkstyle/blob/master/config/spotbugs-exclude.xml#L82

Reply:
Sorry sir, I don't quite understand what is the mean of "**append such case to existing group**".

Reply:
Review this file , there is suppression already for this rule. Append your case to existing suppressions.

Reply:
done.